include/openssl/core_names.h

```diff
@@ -65,6 +65,7 @@ extern "C" {
 
 /* cipher parameters */
 #define OSSL_CIPHER_PARAM_PADDING              "padding"      /* uint */
+#define OSSL_CIPHER_PARAM_USE_BITS             "use-bits"     /* uint */
```

Comment:
Is a more descriptive name possible here?  `"bit-mode"`?
Just the string, the macro name is okay.

Reply:
Or `length-in-bits`? But then it is still unclear which length.

Reply:
Or `length-is-bits`

Reply:
`bit-at-a-time`

Reply:
All those names seem to suck :)

Reply:
More or less than the incumbent?

Reply:
about the same..